src/docs/sphinx/dev_guide/tensor_dual.rst

```diff
@@ -0,0 +1,405 @@
+.. _header-n0:
+
+============
+Tensor Class
+============
+
+``tensor`` is a class template for doing arithmetic on small,
```

Comment:
Can you add a link to the doxygen class page at the top?

Reply:
How do I find the doxygen entries for this branch?

Reply:
https://serac.readthedocs.io/en/feature-variational_form/doxygen/html/structserac_1_1tensor_3_01T_00_01first_00_01rest_8_8_8_01_4.html

That's just a random class i clicked on for your branch.  No guarantee its the right one.

Reply:
Another dumb doxygen question: there are a lot of free functions associated with tensors (arithmetic operators, `det`, `inv`, `tr`, etc), is there a way to get those to appear on the doxygen page along with the `tensor` class? As is, the doxygen documentation for `tensor` makes it look like it doesn't really do anything!

Reply:
not exactly what you are asking for but you might be able to do something with [grouping](https://www.doxygen.nl/manual/grouping.html)

Reply:
@samuelpmishLLNL I believe it's a doxygen bug. The workaround I found to be most convenient is to doxygen document the namespace. The free methods should then appear.